usl_pipeline/usl_lib/usl_lib/transformers/polygon_transformers.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional, Tuple
+
+import numpy
+import numpy.typing as npt
+import rasterio
+from rasterio import features
+from shapely import geometry
+
+from usl_lib.shared import geo_data
+
+
+def rasterize_polygons(
+    header: geo_data.ElevationHeader,
+    polygons_with_masks: list[Tuple[geometry.Polygon, int]],
+    background_value: int = 0,
+) -> Optional[npt.NDArray[numpy.float64]]:
+    """Rasterize polygons to an integer raster matrix.
+
+    Args:
+        header: Coordinate system and raster grid information.
+        polygons_with_masks: Polygons to rasterize with assigned mask values (mask
+            values are used to draw and fill in polygon shape in the raster).
+        background_value: Optional background mask used to fill in the raster before
+            drawing polygons (default value is 0).
+    """
+    if len(polygons_with_masks) == 0:
+        return None
```

Comment:
Why return `None` rather than an empty array here?

Reply:
Yeah, you're right. Fixed.